monorepo/monorepo-ws/apps/update-supergraph-app/src/supergraph-config.yml

```diff
@@ -1,9 +1,9 @@
 subgraphs:
   users:
-    routing_url: http://localhost:8080/graphql
+    routing_url: http://userapp:8080/graphql
```

Comment:
jako tohle je dobrej přístup, ale podchycuje jen nějakej konkrétní compose případ. v produkci by takovej config nemohl fungovat. takže minimálně bych ho pojmenoval podle nějakého prostředí. a nevim jestli to tam máš, ale v readme bych uvedl jak vytvořit config pro další prostředí.

Reply:
pořešíme v dalším pull requestu